hypothesis-python/src/hypothesis/extra/numpy.py

```diff
@@ -729,7 +733,7 @@ def nested_dtypes(
 @st.defines_strategy()
 @deprecated_posargs
 def valid_tuple_axes(
-    ndim: int, *, min_size: int = 0, max_size: int = None
+    ndim: int, *, min_size: int = 0, max_size: Optional[int] = None
```

Comment:
I'd find this easier to read with one argument per line now.

Reply:
Unfortunately the line fits the characters limit and black formats it the way it is. However, I added a tailing comma so that when you update the black version to the `20.8b1` it will automatically enforce splitting arguments over multiple lines.